src/main/java/com/salesforce/dynamodbv2/mt/util/StreamsRecordCache.java

```diff
@@ -190,7 +190,9 @@ Segment subSegment(BigInteger from, BigInteger to) {
          * @return Index in list for given sequence number.
          */
         private int getIndex(BigInteger sequenceNumber) {
-            final List<BigInteger> sequenceNumbers = Lists.transform(records, StreamShardPosition::at);
+            final List<BigInteger> sequenceNumbers = records.stream()
```

Comment:
Guava encourages migrating off of `Lists.transform`

Reply:
I intentionally used Lists.transform here, because it’s lazy, so only sequence numbers that binary search looks at are parsed into BigIntegers. With this change we iterate and parse all entries.